### PR TITLE
enhance dynunet docstrings

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Signed-off-by: Yiheng Wang <vennw@nvidia.com>

### Description
Explain the relationship pf output size and strides for DynUNet .

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/networks/nets/ahnet.py [hunk 1]
```diff
@@ -318,8 +318,8 @@ class AHNet(nn.Module):
     ``"transpose"`` rather than linear interpolations is faster. Therefore, this implementation sets ``"transpose"``
     as the default upsampling method.
 
-    To meet to requirements of the structure, for ``transpose`` mode, the input size of the first ``dim-1`` dimensions  should
-    be divisible by 2 ** (psp_block_num + 3) and no less than 32. For other modes, the input size of the first
+    To meet to requirements of the structure, for ``transpose`` mode, the input size of the first ``dim-1`` spatial dimensions
```
Comment (Nic-Ma) [inline monai/networks/nets/ahnet.py:321]: May I know what the `dim-1` means here?

Thanks.

**Review by Nic-Ma — APPROVED** [review]

File: monai/networks/nets/ahnet.py (hunk 1 above)
Comment (wyli) [inline monai/networks/nets/ahnet.py:321]:
```suggestion
    To meet the requirements of the structure, for ``transpose`` mode, the input size of the first ``dim-1`` spatial dimensions
```

Comment (yiheng-wang-nv) [reply monai/networks/nets/ahnet.py]: Hi Nic, here if `spatial_dims=3`, `dim-1` means 2. I changed the description to avoid confusions.

Comment (Nic-Ma) [reply monai/networks/nets/ahnet.py]: ok, looks good to me.
Thanks.

**Pull request merged**